### DISCUSSION TITLE
There should be a disable command.

### DISCUSSION THREAD

**Discussion by Juhroo** [Zrips/CMI#5066] [Feature Requests]
Like if you go in alias

**/balance: boolean**

There should be a

**/cmi balance: boolean**

So if /cmi balance is false. Player cant do /cmi balance but rather they can do /balance only

**Comment by CloudeLecaw** [comment]
uff
since /balance is just the alias of the real command /cmi balance i dont know if this is really possible

u want that a play cant execute the real command
but can execute the alais

this would be nice for many many plugins where player dont use the alias
but this is not possible to add, it would need a whole recode of command how they are. and also take away the sense of alias

So what is your problem that players should not use /cmi balance?
Well if you use a commandspy of cmi u may noticed allready - each time a player uses /balance the plugin allways just sees the real command /cmi balance and wrote this for you in chat.
u need to look into console to see what they really used.

**Comment by broken1arrow** [comment]
> uff
> since /balance is just the alias of the real command /cmi balance i dont know if this is really possible
> 
> u want that a play cant execute the real command
> but can execute the alais
> 
> this would be nice for many many plugins where player dont use the alias
> but this is not possible to add, it would need a whole recode of command how they are. and also take away the sense of alias
> 
> So what is your problem that players should not use /cmi balance?
> Well if you use a commandspy of cmi u may noticed allready - each time a player uses /balance the plugin allways just sees the real command /cmi balance and wrote this for you in chat.
> u need to look into console to see what they really used.

He probably not want players to know with plugin he use.

But he should deny perms for use balance and use custom alias and use asconsole instead.

**Comment by CloudeLecaw** [comment]
Jep the simple way here is
To deny the permission
And create a custom alias that can use the bperm condiction (bypass that player does not have permission)
Thus custom alais call it balance.
Dont forget to disable Balance in alias.yml

Done

Hm not letting know someone u use CMI, worldgaurd, or other bigger Plugins is total nonsense. There is alawas a way cant block the whole cmi command itself from tab.
U have nearly no chance. Latestly when they will type /cmi they will see it in tab ;)

You should think about your stratgiena bit in that case "I dont want Player know i use CMI,WE,WG,VAULT...etc...  Nonsense
Better keep track of that they cant missuse your plugin with lags or by wrong permission Setup ;)

**Comment by broken1arrow** [comment]
> Jep the simple way here is
> To deny the permission
> And create a custom alias that can use the bperm condiction (bypass that player does not have permission)
> Thus custom alais call it balance.
> Dont forget to disable Balance in alias.yml
> 
> Done
> 
> Hm not letting know someone u use CMI, worldgaurd, or other bigger Plugins is total nonsense. There is alawas a way cant block the whole cmi command itself from tab.
> U have nearly no chance. Latestly when they will type /cmi they will see it in tab ;)
> 
> You should think about your stratgiena bit in that case "I dont want Player know i use CMI,WE,WG,VAULT...etc... Nonsense
> Better keep track of that they cant missuse your plugin with lags or by wrong permission Setup ;)

The are server owners some not like players know what plugin they use (so you not copy it to your own server), you have eZProtector for example you can block tab-complete and commands.

**Comment by CloudeLecaw** [comment]
eZProtector...
I renember i used that years ago when i started with own mc server,bbefore it Stopped working. It had a lot of bug..but ok, its not usefull for newer servers.

TodaY u can find out pluglists very fast without needed to play on a server. Used plugins are not relevant.
The configuration/setup of that is the mystic that normaly noone can see and that creates that what the players will renember

Anl tool that blocks the main command is not needed innmy eyes

**Comment by CloudeLecaw** [comment]
u know request here got ignored by the devs mostly? This here is jsut for issues
send them here aggain: https://github.com/Zrips/CMI/discussions
